examples/ActionGroup/ActionGroup.Tests.ps1

```diff
@@ -2,31 +2,69 @@ BeforeAll {
   Import-Module Az.InfrastructureTesting
 }
 
+$resourceType = "ActionGroup"
+$resourceName = "sampleaction"
+$rgName = "rg-test"
+
+Describe 'Verify Action Group with Confirm-AzBPResource' {
+  it 'Should contain an actiongroup named sampleaction' {
```

Comment:
Looking at this, it makes me wonder if this should have it's own `Describe` block, or if the `It` blocks should be grouped with the other tests as appropriate. `it 'Should contain an action group named simpleaction - ConfirmAzBPresource'` within `Describe 'Verify Action Group Exists'` block. With the same organization for all `Test` files.